### PR TITLE
migration status tracker

### PR DESCRIPTION
**Pull request by iwolkov**
Migration status tracker added.

### CODE REVIEW

File: ide-migration/migration-statistic.html
Comment (StanZGenchev) [inline ide-migration/migration-statistic.html:38]: Replace with `<link rel="icon" sizes="any" href="data:;base64,iVBORw0KGgo=">`

**Review by StanZGenchev — APPROVED** [review]

File: ide-migration/server/migration/api/track-service.js
Comment (ThuF) [inline ide-migration/server/migration/api/track-service.js:101]: isn't `entryInstance` equals to `entry.id`? If that's the case, then t he `migrationsTable.find()` is not needed.

**Review by ThuF — APPROVED** [review]
+1

**Comment by ThuF** [comment]
@iwolkov should we merge the PR? Are we waiting for more changes?

**Comment by StanZGenchev** [comment]
@ThuF No, we are waiting for approval from @vmutafov

File: ide-migration/server/migration/api/track-service.js
Comment (iwolkov) [reply ide-migration/server/migration/api/track-service.js:101]: Yes, you are right

**Review by ThuF — APPROVED** [review]

**Pull request merged**
